script/main.js

```diff
@@ -0,0 +1,33 @@
+var mainPage = document.querySelector('.main');
+var profilePage = document.querySelector('.profile');
+var signInPage = document.querySelector('.signin');
+
+
+var switchMain = document.querySelector('.switch-main');
+switchMain.addEventListener('click', function() {
+    router('main')
+})
+
+var switchProfile = document.querySelector('.switch-profile');
+switchProfile.addEventListener('click', function() {
+    router('profile')
+})
+
+var switchSignin = document.querySelector('.switch-signin');
+switchSignin.addEventListener('click', function() {
+    router('signin')
+})
+
+var router = function(page) {
```

Comment:
hey, instead of if else statements, could we use a router pattern i.e. inside of an object?